packages/affiliates/payouts/devmining/2021-03-15_2021-03-22_0019.json

```diff
@@ -6,187 +6,45 @@
+      ["0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037", "0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87", "1.2.2"],
+      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "0x531B290230b1809d0AE29Ea834EdF96748F49Bf9", "1.2.2"],
+      ["0x2862A798B3DeFc1C24b9c0d241BEaF044C45E585", "0xa1eD088d0789e7baBa18Ed4d87baca65376850d8", "1.2.2"],
+      ["0xd81028a6fbAAaf604316F330b20D24bFbFd14478", "0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc", "1.2.2"],
+      ["0x7c4090170aeADD54B1a0DbAC2C8D08719220A435", "0x7eb1e51351380E67F641B84545bf41eCdc10298E", "1.2.2"],
+      ["0xaD3cceebeFfCdC3576dE56811d0A6D164BF9A5A1", "0x7eb1e51351380E67F641B84545bf41eCdc10298E", "1.2.2"],
+      ["0xC843538d70ee5d28C5A80A75bb94C28925bB1cf2", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "1.2.2"],
+      ["0xeFA41F506EAA5c24666d4eE40888bA18FA60a1c7", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "1.2.2"],
+      ["0xfA3AA7EE08399A4cE0B4921c85AB7D645Ccac669", "0x97990B693835da58A281636296D2Bf02787DEa17", "1.2.2"],
+      ["0x9E929a85282fB0555C19Ed70942B952827Ca4B0B", "0x1Da788aBB922325E0ea2533019bf09f8a7525AC0", "2.0.1"],
+      ["0x6DA66C15823cFf681DaD6963fBD325a520362958", "0x1Da788aBB922325E0ea2533019bf09f8a7525AC0", "2.0.1"],
+      ["0xD50fbace72352C2e15E0986b8Ad2599627B5c340", "0x9C8F094985d12d80e930d2abBB6471EDF68d3bEB", "1.2.2"],
+      ["0x45788a369f3083c02b942aEa02DBa25C466a773F", "0xDAa953f01048253A201BA9B0bd0786575f9C2468", "2.0.1"],
+      ["0x0388f65C185a7E7D857BB142185381d97a4bc747", "0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c", "2.0.1"],
+      ["0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D", "0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c", "2.0.1"],
+      ["0x14A415Dd90B63c791C5dc544594605c8bC13Bc8D", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0x1066E9D2E372d01A0F57bB6f231D34Ce4CEd228e", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0xa24Ba528Be99024f7F7C227b55cBb265ecf0C078", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0xd60139B287De1408f8388f5f57fC114Fb4B03328", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0x8E51Ad4EeB19693751a9A3E36b8F098D891Ddc7f", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E", "2.0.1"],
+      ["0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43", "0x97990B693835da58A281636296D2Bf02787DEa17", "2.0.1"],
+      ["0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C", "0x97990B693835da58A281636296D2Bf02787DEa17", "2.0.1"],
+      ["0x964Be01cCe200e168c4ba960a764cBEBa8C01200", "0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71", "2.0.1"],
+      ["0xDB2E7F6655de37822c3020a8988351CC76caDAD5", "0xAd2B08F878C0Be4C4162A72bbF7c0f547F93bD9f", "2.0.1"],
+      ["0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a", "0x2CA781A4dcA1f37F52696238e70A004746068F3b", "2.0.1"],
+      ["0xb40BA94747c59d076B3c189E3A031547492013da", "0xd332f9Faf23Ba97d9f0f9EE9b3d8798C49Ff269B", "2.0.1"]
     ],
     "fallbackPrices": [
```

Comment:
What are these?

Reply:
this was a previous payout record which was saved to the wrong directory and not linted. i moved and linted it